build.gradle

```
@@ -24,9 +24,16 @@ subprojects {
 
     tasks.named("dokkaHtml", org.jetbrains.dokka.gradle.DokkaTask.class) {
         outputDirectory.set(file("${rootDir}/reference-docs"))
+
+        String dokkaBaseConfiguration = """
+        {
+          "footerMessage": "<a href=\\"https://www.joinforage.com/\\"style=\\"color: inherit; text-decoration: none;\\">© 2023–2025 Forage Technology Corporation.</a>",
```

```suggestion
        "footerMessage": "© 2023–2025 <a href=\\"https://www.joinforage.com/\\" style=\\"color: inherit; text-decoration: none;\\">Forage Technology Corporation</a>.",
```
Better to have the link only be the "Forage Technology Corporation" part of the text.

Tested this config change, and the docs generate great. Please update / merge!

---

oh shoot, you meant the first update and then merge. I read it as merge, so I did not include this - apologies. We can either forget about it or @pixelitobenito you can open another tiny PR for the fix. Just let me know

---

I made the change so you just merged it in :)